system/core-workflows/how-do-they-work.rst

```diff
@@ -20,72 +19,71 @@ This will decide on your available conditions and actions.
 
 .. tip::
 
-   You will be able to use objects that are in relation to your selection in
+   You will be able to use attributes that are in relation to your selection in
    your conditions.
-   
+
    | This means:
    | Ticket objects also have access to the ticket customer.
 
 Context
 -------
 
 Choose in which situation the workflow is applied.
-Contexts can be combined to reduce workflows.
+Contexts can be combined to avoid duplicate workflows.
 
 Creation mask
-   Once selected your conditions and actions will affect all applicable creation
+   If selected, your conditions and actions will affect all applicable creation
    masks.
 
 Edit mask
-   Once selected your conditions and actions will affect all applicable edit
+   If selected, your conditions and actions will affect all applicable edit
    masks.
 
 Conditions
 ----------
 
-Zammad decides in between selected and saved conditions.
+Zammad differentiates between selected and saved conditions.
 These can be combined wherever needed.
 
-   .. tip:: **🤓 Combining conditions allows "OR"-selections**
-
-      However, note that each condition type counts as *and* selector
-      and can't overrule the other condition type.
-
-      Every attribute can be used once per condition type.
+   .. warning:: **⚠️ Restrict workflows to specific roles if needed!**
 
-   .. warning:: **⚠ Restrict on role basis if needed ⚠**
-
-      By default, unless configured in conditions, workflow rules are
-      evaluated for **all roles**. This also affects your customers! 🙀
+      By default and unless configured in conditions, workflow rules are
+      executed for **all roles**. This also affects your customers!
 
 Selected Conditions
-   These conditions only match if they're active in selection.
-   This applies for drafts (active selection) and currently saved values.
+   These conditions only match if a complying selection is made (e.g. choosing
+   another group in the ticket without saving).
+   This applies also for the the creation of items and temporarily saved values.
```

Comment:
the old wording was a bit more correct.

Selected conditions are based on the current form values (unsaved changes) on top of the currently saved values of the ticket as basement if the ticket is already stored.

Reply:
I changed the wording of the block. If it is still not good, please let me know.